### PR TITLE
Modify DB populate script with enough tuples to fulfill the functional spec requirements

### PR DESCRIPTION
**Pull request by spacebugs**
Finished script to populate all tables. This includes players so you may have to have a fresh db before running it.

### CODE REVIEW

File: server/sql/populate_db.sql [hunk 1]
```diff
@@ -1,12 +1,1083 @@
 -- Populate DB with some preset values for testing
 -- Since this is just used for testing, don't expect many of these numbers to make sense in a real game
 
--- This won't work if run directly, it must be run through the go test script (for proper password hashing)
+INSERT INTO Player VALUES
+('ae1', 'password1', 'Albert Einstein'),
```
Comment (knsjerome) [inline server/sql/populate_db.sql:5]: Plain-text passwords should never be stored. Please go through the effort to getting these passwords hashed via the Go code and then you can copy-paste the hashed values into this insert statement.

**Review by knsjerome — CHANGES_REQUESTED** [review]
Hash the passwords before storing them

File: server/sql/populate_db.sql (hunk 1 above)
Comment (ksmolko) [reply server/sql/populate_db.sql:5]: One other thing: If we are going to paste the hashed passwords into the SQL script here, then we should remove the following lines here: https://github.com/knsjerome/cmpt-354-project/blob/master/server/tests.go#L22-L26

File: server/sql/populate_db.sql [hunk 2]
```diff
@@ -311,6 +1675,14 @@ INSERT INTO CampaignMilestones VALUES
 (
     2,
     'Started your wonderful journey'
+)
```
Comment (ksmolko) [inline server/sql/populate_db.sql:1678]: Should have a comma here

File: server/sql/init_db.sql [hunk 3]
```diff
@@ -298,7 +298,7 @@ CREATE TYPE e_fighting_style AS ENUM (
     'Archery',
     'Blind Fighting',
     'Defense',
-    'Druidic Warrior'
+    'Druidic Warrior',
```
Comment (ksmolko) [inline server/sql/init_db.sql:301]: Oops thanks for catching this lol

**Review by ksmolko — APPROVED** [review]

**Review by knsjerome — APPROVED** [review]

**Pull request merged**